SYCL/ESIMD/ext_math.cpp

```diff
@@ -28,7 +28,6 @@
 
 using namespace cl::sycl;
 using namespace sycl::ext::intel;
-using namespace sycl::ext::intel::esimd;
```

Comment:
I wonder if we can use something like this (not sure about the proper spelling)

```
  template <class T>
  using simd = esimd::simd<T>
```

to eliminate  the changes below.

Reply:
I think using `esimd::simd<T, N>` in the code below is clearer compared to introducing `simd` alias.

Reply:
I think using `esimd::simd<T, N>` in the code below is clearer compared to introducing `simd` alias.

Reply:
I agree with @kbobrovs I think `esimd::simd<T, N>` is more clear.